core/rawdb/accessors_chain_test.go

```diff
@@ -291,10 +291,10 @@ func TestBlockReceiptStorage(t *testing.T) {
 	// Insert the receipt slice into the database and check presence
 	WriteReceipts(db, hash, 0, receipts)
 	if rs := ReadReceipts(db, hash, 0, 0, params.TestChainConfig); len(rs) == 0 {
-		t.Fatalf("no receipts returned")
+		t.Fatal("no receipts returned")
```

Comment:
The `else` is unnecessary  because of this `t.Fatal`, which stops the test immediately. Reducing the nesting on the next check improves clarity.

I'd add a suggestion but GitHub sucks for code review and won't allow them on deleted lines.